testing/integration-tests/src/full_client/storage/mod.rs

```diff
@@ -235,3 +235,54 @@ async fn storage_pallet_storage_version() -> Result<(), subxt::Error> {
         .await?;
     Ok(())
 }
+
+#[subxt_test]
+async fn storage_decode_keys() -> Result<(), subxt::Error> {
```

Comment:
Nit: add `iter` to the test name?

```suggestion
async fn storage_iter_decode_keys() -> Result<(), subxt::Error> {
```